timeboost/src/binaries/mkconfig.rs

```diff
@@ -3,17 +3,19 @@ use std::num::NonZeroU8;
 use std::{io, iter};
 
 use anyhow::{Result, bail};
+use ark_std::rand::SeedableRng as _;
 use clap::{Parser, ValueEnum};
 use cliquenet::Address;
 use multisig::x25519;
+use secp256k1::rand::SeedableRng as _;
 use timeboost_crypto::{DecryptionScheme, TrustedKeyMaterial};
 use timeboost_utils::keyset::{KeysetConfig, NodeInfo, PrivateKeys, PublicDecInfo};
 use timeboost_utils::types::logging;
 
 #[derive(Clone, Debug, Parser)]
 struct Args {
     /// How many nodes should configuration contain?
-    #[clap(long, short)]
+    #[clap(long, short = 'n')]
```

Comment:
Isn't `n` implied by `short`?

Reply:
yes you are right, reverted in [722c69f](https://github.com/EspressoSystems/timeboost/pull/357/commits/722c69fed6e246cb460cb178158c81fbf3d29b4f)